### PR TITLE
Allow missing values in other realizations than current.

### PR DESCRIPTION
**Pull request by berland**
Improve warning message pointing to empty cells

Solves #234 and #235

### CODE REVIEW

File: tests/jobs/design2params/test_design2params.py [hunk 1]
```diff
@@ -304,7 +323,7 @@ def test_existing_parameterstxt(
 def test_open_excel_file_wrong_defaults(input_data):
```
Comment (berland) [inline tests/jobs/design2params/test_design2params.py:323]: (I don't think this test is doing what its name suggests)

Comment (berland) [reply tests/jobs/design2params/test_design2params.py:323]: as it stands, it can be deleted without loosing test coverage.

Comment (berland) [reply tests/jobs/design2params/test_design2params.py]: Changed.

**Review by oysteoh — APPROVED** [review]
👍

**Pull request merged**